objectivec/test/ort_session_test.mm

```diff
@@ -191,6 +192,28 @@ - (void)testRunFailsWithInvalidInput {
   ORTAssertBoolResultUnsuccessful(runResult, err);
 }
 
+- (void)testAppendCoreMLEP {
+  NSError* err = nil;
+  ORTSessionOptions* sessionOptions = [ORTSessionTest makeSessionOptions];
+  ORTCoreMLExecutionProviderOptions* coreMLOptions = [[ORTCoreMLExecutionProviderOptions alloc] init];
+
+  BOOL appendResult = [sessionOptions appendCoreMLExecutionProviderWithOptions:coreMLOptions
+                                                                         error:&err];
+
+  if (!ORTIsCoreMLExecutionProviderAvailable()) {
```

Comment:
Is there a test where we set one of the options?

Reply:
I could set some option values - not sure how to check the effects of doing that though. Open to suggestions for this.

Reply:
Probably fine to manually validate in a debugger for now if there's no better way. 

Just wanted to make sure things work as expected and that there was example code of actually setting an option.